common/persistence/dataManagerInterfaces.go

```diff
@@ -266,6 +266,12 @@ type (
 		Msg string
 	}
 
+	// CassandraLWTError is returned when we get UNAVAILABLE error code from Cassandra with message containing
+	// "Cannot perform LWT operation"
+	CassandraLWTError struct {
```

Comment:
We should keep this file DB-agnostic, so I'd prefer to use more generic names like DBUnavailableError which could be meaningful for plugins such as MongoDB.

Also see my similar comment in `common/persistence/nosql/nosqlplugin/interfaces.go`.